src/tests/components/searchForm.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
The filename should match that of the component under test, including the casing style (so SearchForm.test.js)